index.html

```diff
@@ -994,6 +994,79 @@
+
+Similarly, unexpected profiling is related to [[[#hl-sensitive-information]]],
+but the former is a superset of the latter;
+all cases of unexpected sensitive information disclosure are examples of
+unexpected profiling, but Web users may have attributes or characteristics
+about themselves that are not universally thought of as "sensitive", but which
+they never the less do not wish to share with the sites they visit. People
+may wish to not share these "non-sensitive" characteristics
+for a variety of reasons (e.g., a person may worry that their ideas of what
+counts as "sensitive" is different from others, a person might might be ashamed
+or uncomfortable about a character trait or they might simply not wish to be profiled).
+
+Profiling occurs for many reasons. It can be used to facilitate price
+discrimination or offer manipulation, to make inferences about what products or
+services users might be more likely to purchase, or more generally, for a site
+to learn attributes about the Web user the Web user does not intend to share.
+Unexpected profiling can also contribute to feelings of powerlessness and
+loss of agency.
+  
+A privacy hardm occurs if the user reasonably expects that they will not be profiled or singled-out when they visit a site, but the site does it anyway.
+  
+See the W3C TAG Finding on [Unsanctioned Web Tracking](https://www.w3.org/2001/tag/doc/unsanctioned-tracking/) for more information on why Unsanctioned Web Tracking is harmful for the Web.
+
+A user might expect that their visit will not cause them to be profiled or singled-out if they:
+* Use a browser that offers tracking protection
+* Use extensions or add-ons to block tracking
+*	Choose no, deny or disallow when offered a choice about tracking cookies (or tracking in general)
+*	Use their browser's private or incognito browsing mode
+*	Use a clean browser profile, container, partition or other mode to isolate web site visits
+*	Explicitly clear the site's cookies or storage
+* Use a VPN to hide their location
+* Where they believe privacy or data protection laws would not permit profiling
+
+Ironically, some of those strategies could reveal attributes or characteristics that would lead
+unexpected profiling, were a site able to learn them such as:
+
+* Whether a person is using a VPN to hide their location
+* Whether a person is using a clean browser profile, container, partition or
+  other mode to isolate web site visits
+* Demographic information, even in the presence of other privacy protections
+  (e.g., k-anonymity, differential privacy)
+
+<div class="example">
+
+Peter is a <a href="https://en.wikipedia.org/wiki/Furry_fandom">furry</a>.
+Despite knowing that there are thousands of
+other furries on the internet, and despite using a browser with
+robust browser fingerprinting protections, and despite the growing cultural
+acceptance of furries, Peter does not want (most) sites to learn
+or personalize content around his furry-interest.
+
+</div>
```

Comment:
I think that the example could be clarified. If we make this section about unexpected inferences more specifically, then Peter can have an account with that site and interact with it often, but not expect that his regular reading of articles about foxes and synthetic fur will have him profiled as a furry. When he starts getting ads for furry atire, he feels his privacy has been violated.

Reply:
I dont intend this text to be about unexpected inferences (maybe profiling is the wrong word then?).  My goal in this example, and in the section in general, is to establish that its privacy-harmful to unexpectedly share information about me / a user, independent of whether 1) that information is "sensitive" by some definition, or 2) that information allows me to be (re)identified, as myself or pseudonymously